### PR TITLE
Release notes for 15.5.0

### PR DESCRIPTION
**Pull request by harshjain2**
Release notes for 15.5.0

### CODE REVIEW

File: docs/releases.md [hunk 1]
```diff
@@ -1,5 +1,32 @@
...
+* Added error message and help when vstest.console is invoked without arguments.
+* Fixed failure in loading extensions without Identifier Data.
+* Handled Test Host close.
+* TestCase Display Name is displayed instead of FullyQualifiedName.
+* Fixed issues with Static Cover Coverage, Ordered tests through TMI.
+
+### New Features introduced
+* Added Telemetry Infra for Design Mode.
+* Supported running .Net Framework v35 in compat mode.
+* Loclization changes.
```
Comment (sudiptadmsft) [inline docs/releases.md]: Localization

File: docs/releases.md [hunk 2]
```diff
@@ -1,5 +1,32 @@
...
+* Automatically find Platform and Framework if not specified explicitly.
+* Changes for making Test Platform as default in Visual Studio.
```
Comment (sudiptadmsft) [inline docs/releases.md]: Not a feature

File: docs/releases.md (hunk 1 above)
Comment (pvlakshm) [reply docs/releases.md]: fix typo in localization, please.

File: docs/releases.md (hunk 2 above)
Comment (pvlakshm) [reply docs/releases.md]: Making it the default is a not feature per se. Remove that line.

**Pull request merged**